pkg/apis/serving/v1alpha2/inferenceservice_conversion.go

```diff
@@ -89,7 +89,6 @@ func (src *InferenceService) ConvertTo(dstRaw conversion.Hub) error {
 		}
 	} else if src.Spec.Default.Predictor.PyTorch != nil {
 		dst.Spec.Predictor.PyTorch = &v1beta1.TorchServeSpec{
-			ModelClassName: src.Spec.Default.Predictor.PyTorch.ModelClassName,
```

Comment:
I think we still need this as v1alpha2 always converts to kfs pytorchserver with v1 protocol.